### DISCUSSION TITLE
Use LandCover.ai model as checkpoint for own model

### DISCUSSION THREAD

**Discussion by keepcool13** [PUTvision/qgis-plugin-deepness#181] [Q&A]
Hi! I'm trying to detect landuse from an aerial image, and the LandCover.ai model provided in your zoo works ok. But I think it could be improved with some training on my own dataset.

Ideally, I would also need slightly different categories than in your model:

- buildings
- roads
- green area (not only woodland, but also fields, parks...)
- other sealed surface (e.g. parking lots, backyards)

What would be the best way to do it? Does it make sense to fine-tune your model, or use it as a checkpoint? (Does this mean the same or not? I'm quite new to the field of ML, so not sure about the terms). Or do you think it isn't worth the effort, and I should rather train the model from scratch?

**Comment by przemyslaw-aszkowski** [comment]
Hi,

I think it would be the easiest to start from a new model, as our is in ONNX format, and may require more time to use it as a checkpoint. Nevertheless, you can include the data we used as a pretraining step for your model. There are more details provided in the documentation: https://qgis-plugin-deepness.readthedocs.io/en/latest/example/example_segmentation_landcover.html

Cheers
Przemek
